### PR TITLE
Run aspell on the documentation

### PR DESCRIPTION
**Pull request by jonas**

### CODE REVIEW

File: docs/src/main/paradox/java/http/implications-of-streaming-http-entity.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ eagerly consume the entity and make it available in memory:
 ### Discarding the HTTP Response Entity (Client)
 
 Sometimes when calling HTTP services we do not care about their response payload (e.g. all we care about is the response code),
-yet as explained above entity still has to be consumed in some way, otherwise we'll be exherting back-pressure on the
+yet as explained above entity still has to be consumed in some way, otherwise we'll be exerting back-pressure on the
```
Comment (jonas) [inline docs/src/main/paradox/java/http/implications-of-streaming-http-entity.md]: Not sure about this one!

**Comment by ktoso** [comment]
Did you use US or UK english? We tend to use US english (though I find that weird being UK biased ;-))

**Comment by ktoso** [comment]
Awesome idea btw

File: docs/src/main/paradox/scala/http/implications-of-streaming-http-entity.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ eagerly consume the entity and make it available in memory:
 ### Discarding the HTTP Response Entity (Client)
 
 Sometimes when calling HTTP services we do not care about their response payload (e.g. all we care about is the response code),
-yet as explained above entity still has to be consumed in some way, otherwise we'll be exherting back-pressure on the
+yet as explained above entity still has to be consumed in some way, otherwise we'll be exerting back-pressure on the
```
Comment (jonas) [inline docs/src/main/paradox/scala/http/implications-of-streaming-http-entity.md]: ... also found here

**Comment by jonas** [comment]
I used US, but did not convert British to American spelling: specialised => specialized etc.

**Comment by jonas** [comment]
If I get time I might try to come up with some scripts to run aspell or hunspell with a pregenerated dictionary to reduce noise from code snippets etc.

**Comment by jonas** [comment]
Failure is same as #333 which was supposed to be resolved by #403. Reopen?

**Comment by ktoso** [comment]
When did you branch off this PR? Can you rebase just to be sure that fix is included here?

**Comment by jonas** [comment]
I branched on top of current master
```
> git branch --contains 3f807a71182378effd0ce89bb2ad68ed8934f67f
  master
* spell-checking
> git log --graph -n 5 --oneline
* 30ce2ff Run aspell on the documentation
* 66adf84 fix references to http-experimental (#406)
* b0300b3 dont publish tests, in proper style (#404)
* c258990 Put akka-stream-testkit in test configuration (#401) (#402)
* 3f807a7 =htp #333 increase timeout, 1 second is too little on jenkins (#403)

**Comment by jonas** [comment]
Rechecked and it is a new failure for case with seven parameters.

https://jenkins.akka.io:8498/job/pr-validator-akka-http/160/consoleFull
```
[info] - should properly bind and handle a server with a flow (with seven parameters) *** FAILED ***
[info]   java.util.concurrent.ExecutionException: akka.stream.StreamTcpException: Tcp command [Connect(localhost:42995,None,List(),Some(10 seconds),true)] failed
[info]   at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:357)
[info]   at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1907)
[info]   at scala.concurrent.java8.FuturesConvertersImpl$CF.scala$concurrent$java8$FuturesConvertersImpl$CF$$super$get(FutureConvertersImpl.scala:84)
[info]   at scala.concurrent.java8.FuturesConvertersImpl$CF$$anonfun$get$2.apply(FutureConvertersImpl.scala:84)
[info]   at scala.concurrent.BlockContext$DefaultBlockContext$.blockOn(BlockContext.scala:53)
[info]   at scala.concurrent.package$.blocking(package.scala:123)
[info]   at scala.concurrent.java8.FuturesConvertersImpl$CF.get(FutureConvertersImpl.scala:84)
[info]   at akka.http.javadsl.HttpExtensionApiSpec.akka$http$javadsl$HttpExtensionApiSpec$$waitFor(HttpExtensionApiSpec.scala:474)
[info]   at akka.http.javadsl.HttpExtensionApiSpec$$anonfun$1$$anonfun$apply$mcV$sp$7.apply(HttpExtensionApiSpec.scala:155)
[info]   at akka.http.javadsl.HttpExtensionApiSpec$$anonfun$1$$anonfun$apply$mcV$sp$7.apply(HttpExtensionApiSpec.scala:145)
[info]   ...
[info]   Cause: akka.stream.StreamTcpException: Tcp command [Connect(localhost:42995,None,List(),Some(10 seconds),true)] failed
```

**Comment by jonas** [comment]
Could the test failure be caused by a race condition in how TestUtil finds available ports and the test binds to it? Let me know if I should create an issue.

**Comment by ktoso** [comment]
We'll work on fleaky tests separately from this issue, I'll open tickets for each failure.

**Comment by ktoso** [comment]
LGTM, thanks!

**Pull request merged**